example/dj/apps/issue_tracker/tests/ui_permissions.py

```diff
@@ -1,127 +1,103 @@
-
-        CHANGED_USERNAME = 'changed_nick'
-        self.post('%s%s/' % (self.USER_UI_URL, user.pk), data={'edit-is-user-username': CHANGED_USERNAME})
-        assert_http_ok(resp)
-        assert_equal(User.objects.get(pk=user.pk).username, CHANGED_USERNAME)
-
-    @login(is_superuser=False)
-    def test_only_superuser_may_edit_user(self):
-        user = self.get_user_obj()
-        resp = self.get('%s%s/' % (self.USER_UI_URL, user.pk))
-        assert_http_forbidden(resp)
-
-        CHANGED_USERNAME = 'changed_nick'
-        self.post('%s%s/' % (self.USER_UI_URL, user.pk), data={'edit-is-user-username': CHANGED_USERNAME})
-        assert_http_forbidden(resp)
-        assert_not_equal(User.objects.get(pk=user.pk).username, CHANGED_USERNAME)
-
     @login(is_superuser=False)
-    def test_user_may_edit_itself(self):
-        user = self.logged_user.user
-        resp = self.get('%s%s/' % (self.USER_UI_URL, user.pk))
-        assert_http_ok(resp)
-
-        CHANGED_USERNAME = 'changed_nick'
-        self.post('%s%s/' % (self.USER_UI_URL, user.pk), data={'edit-is-user-username': CHANGED_USERNAME})
+    def test_home_view_should_return_ok_for_superuser(self):
+        resp = self.get('/')
         assert_http_ok(resp)
-        assert_equal(User.objects.get(pk=user.pk).username, CHANGED_USERNAME)
 
     @login(is_superuser=True)
-    def test_superuser_may_add_user(self):
-        USERNAME = 'new_nick'
-
-        resp = self.post('%sadd/' % self.USER_UI_URL, data={'add-is-user-username': USERNAME,
-                                                            'add-is-user-password': 'password'})
-        assert_http_redirect(resp)
-        assert_true(User.objects.filter(username=USERNAME).exists())
+    def test_superuser_should_do_all_operations(self):
+        issue = self.create_issue()
+        user = self.create_user('new_user', 'password', 'test@email.com')
+
+        # List
+        assert_http_ok(self.get('/user/'))
+        assert_http_ok(self.get('/issue/'))
+
+        # Add
+        assert_http_ok(self.get('/user/add/'))
+        assert_http_ok(self.get('/issue/add/'))
+        assert_http_ok(self.post('/user/add/', {}))
+        assert_http_ok(self.post('/issue/add/', {}))
+
+        # Detail
+        assert_http_ok(self.get('/user/{}/'.format(user.pk)))
+        assert_http_ok(self.get('/issue/{}/'.format(issue.pk)))
+        assert_http_ok(self.post('/user/{}/'.format(user.pk), {}))
+        assert_http_ok(self.post('/issue/{}/'.format(issue.pk), {}))
 
     @login(is_superuser=False)
-    def test_only_superuser_may_add_user(self):
-        USERNAME = 'new_nick'
-
-        resp = self.post('%sadd/' % self.USER_UI_URL, data={'add-is-user-username': USERNAME,
-                                                            'add-is-user-password': 'password'})
-        assert_http_forbidden(resp)
-        assert_false(User.objects.filter(username=USERNAME).exists())
+    def test_user_without_permission_should_do_nothing(self):
```

Comment:
The test with same method name already exists.

Reply:
but it is in another testcase